plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -1,2 +1,6 @@
 pluginName=Google App Engine Standard Deployment Support UI
```

Comment:
@elharo This comes from the name of the non-UI bundle by appending `UI`, but I can remove it. Or should we remove `Support` from that plugin's name as well?